airflow/dags/parse_and_validate_rt/parse_rt_trip_updates.yml

```diff
@@ -11,7 +11,7 @@ arguments:
   - "trip_updates"
   # we run on a delay because of archiver write latency but then we process the prior _hour_
   - "{{ get_bucket() }}/rt/{{ execution_date.replace(minute=0, second=0).format('YYYY-MM-DDTHH*') }}"
-  - "{{ prefix_bucket('rt-parsed') }}"
+  - "{{ env_var('CALITP_BUCKET__RT_PARSED') }}"
```

Comment:
This in particular is making me a little nervous... It seems like in general we are using the environment variables just for pipeline v2 (= archiver v3) content that:

a) is not yet canonical
b) is clearly separate from v1 (= archiver v2)

I guess for RT are we not going to have separate v2 buckets? It's the grayest area because we already did a lot of the stuff in the original parse jobs.... But wanted to ask

Reply:
I guess specifically I think that this value is maybe set incorrectly in Composer? 
![image](https://user-images.githubusercontent.com/55149902/185226501-5181d67f-30e8-451e-9256-ccc2b58c60b1.png)

Reply:
^ but this is making me wonder if we need to manually (together?) go through each location that needs the environment variable set and make sure it's set correctly

Reply:
Nope this is a great catch; these should not have been changed. I will revise.